### PR TITLE
Parse field type configuration for invalid characters

### PR DESCRIPTION
**Pull request by deanhannigan**
## Description

The field type configuration for the barcode reader was not being parsed correctly. The field `type` string is now stripped of any invalid characters.

Addresses: 
- https://github.com/Budibase/budibase/issues/9523

### CODE REVIEW

File: packages/builder/src/components/design/settings/controls/FormFieldSelect.svelte [hunk 1]
```diff
@@ -25,16 +25,21 @@
...
       types = [type, "field/string"]
     } else {
       types = [type]
     }
 
-    types = types.map(type => type.slice(type.indexOf("/") + 1))
+    types = types.map(type => {
+      let fieldTypeRaw = type.slice(type.indexOf("/") + 1)
+      let fieldTypeParsed = fieldTypeRaw.replace("/", "")
+      return fieldTypeParsed
```
Comment (aptkingston) [inline packages/builder/src/components/design/settings/controls/FormFieldSelect.svelte:38]: Was the problem here that the field type was `field/barcode/qr` but the actual datatype is `barcodeqr`?

Comment (aptkingston) [reply packages/builder/src/components/design/settings/controls/FormFieldSelect.svelte:38]: The reason I ask is just that if it is, we could actually just update the manifest to change the setting type to `field/barcodeqr` couldn't we? Which would just be a one line change and not require and other changes (I think).

Comment (deanhannigan) [reply packages/builder/src/components/design/settings/controls/FormFieldSelect.svelte:38]: That's essentially it. My only concern in modifying the type was any potential side-effect that I might miss. I've run through some more testing and it does seem a manifest update is all that's required here.

**Review by aptkingston — APPROVED** [review]
Nice, glad that worked. LGTM!

**Pull request merged**